src/Controller/ImagineController.php

```diff
@@ -117,7 +117,7 @@ private function getFiltersBc(Request $request): array
             }
         }
 
-        $runtimeConfig = $request->query->get('filters', []);
+        $runtimeConfig = $request->query->all()['filters'] ?? [];
```

Comment:
without this change, these are the errors:

```
 ------ ------------------------------------------------------------------------
  Line   Controller/ImagineController.php
 ------ ------------------------------------------------------------------------
  122    Call to function is_array() with string will always evaluate to false.
  126    Unreachable statement - code above always terminates.
 ------ ------------------------------------------------------------------------
```

`InputBag::get` is supposed to return always `string`.

Reply:
this is the BC part for symfony < 5.1, because the `all` method accepts a parameter only since version 5.1.

imo the right thing here would be to keep the code and ignore the phpstan complaint, with a comment that this is the BC < symfony 5.1 part. due to the version check, we do the right thing.

Reply:
oh sorry, i think i was not clear: i meant to ignore the new warning we get once we tell phpstan that the kernel version is dynamic :sweat_smile:

Reply:
yep, these are the new ones because `InputBag::get` always return `string`.

```
 ------ ------------------------------------------------------------------------
  Line   Controller/ImagineController.php
 ------ ------------------------------------------------------------------------
  122    Call to function is_array() with string will always evaluate to false.
  126    Unreachable statement - code above always terminates.
 ------ ------------------------------------------------------------------------
```

Reply:
ah, its about the is_array now. tricky tricky... 
thanks for the improvements